src/Bot.ts

```diff
@@ -152,4 +155,16 @@ export class Bot extends builder.UniversalBot {
             callback(null, null, 200);
         };
     }
+
+    // set incoming event to any because membersAdded is not a field in builder.IEvent
+    private getMessageReactionsHandler(bot: builder.UniversalBot): (event: any) => void {
+    return async function(event: any): Promise<void> {
```

Comment:
Type this as builder.IMessageUpdate.

Reply:
+1, and don't forget to change return type `(event: any) => void` along with type.